includes/admin/class-admin.php

```diff
@@ -0,0 +1,182 @@
+<?php
+/**
+ * Admin functionality.
+ *
+ * @package AdvancedAds\Genesis
+ * @author  Advanced Ads <info@wpadvancedads.com>
+ * @since   1.0.0
+ */
+
+namespace AdvancedAds\Genesis;
+
+use Advanced_Ads_Admin_Options;
+use AdvancedAds\Framework\Interfaces\WordPress_Integration;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Admin Class.
+ */
+class Admin implements WordPress_Integration {
+
+	/**
+	 * Hook into WordPress.
+	 */
+	public function hooks() {
+		// Add sticky placement.
+		add_action( 'advanced-ads-placement-types', [ $this, 'add_placement' ] );
+
+		// Content of sticky placement.
+		add_action( 'advanced-ads-placement-options-after', [ $this, 'placement_options' ], 10, 2 );
+	}
+
+	/**
+	 * Add placement
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param array $types Hold placements.
+	 *
+	 * @return array $types
+	 */
+	public function add_placement( $types ) {
+		$types['genesis'] = [
+			'title'       => esc_html__( 'Genesis Positions', 'advanced-ads-genesis' ),
+			'description' => esc_html__( 'Various positions for the Genesis theme.', 'advanced-ads-genesis' ),
+			'image'       => AAG_BASE_URL . 'assets/img/genesis.png',
+			'order'       => 81,
+		];
+
+		return $types;
+	}
+
+	/**
+	 * Options for the placement.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $placement_slug Slug of the placement.
+	 * @param array  $placement      Current placement.
+	 *
+	 * @return void
+	 */
+	public function placement_options( $placement_slug = '', $placement = [] ) {
+		// Early bail!!
+		if ( 'genesis' !== $placement['type'] ) {
+			return;
+		}
```

Comment:
In the past, we disallowed yoda conditions because a consistent way of writing conditions provides better clarity.
But it seems like this is a general discussion and nothing you need to change right now.